### DISCUSSION TITLE
post: Create a react Library and Publish to NPM

### DISCUSSION THREAD

**Discussion by giscus** [wghglory/my-website#52] [Announcements]
# post: Create a react Library and Publish to NPM

Guanghui Wang Website

https://derekw.vercel.app/posts/how-to-create-a-react-library

**Comment by irisyuan2019** [comment]
Hi Guanghui, great post! Look forward to the real components you made.

**Reply by wghglory** [reply]
Thank you @irisyuan2019 !

**Comment by jimwilson94** [comment]
Hey bro, long time no see. This is extremely helpful, thanks!

**Reply by wghglory** [reply]
Thank you bro! I'm so glad to hear from you. Please share more feedbacks!
